config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -36,6 +36,38 @@ presubmits:
           - name: ndots
             value: "1"
 
+  - name: pull-cert-manager-make-test
+    always_run: false
```

Comment:
suggestion: Do we not actually want `always_run: true`? I think ideally we'd get to a point where every PR is tested fully with bazel and make, before we remove bazel.

Reply:
You are correct. Since https://github.com/cert-manager/cert-manager/pull/4914 hasn't been merged to master yet, I propose to set the job to optional to prevent any breakage in other PRs.

I have added a TODO in 3aba2cf29694a385bb6b5b2c01fe365acf893ac8.

Reply:
I agree it should be optional currently, but I think it could be `always_run: true` today at least for `make test-ci`. That code _has_ already merged to master. The _e2e_ would have to be `always_run: false` but I think we _can_ always run the tests